tachyon/math/matrix/matrix_utils_unittest.cc

```diff
@@ -21,4 +23,71 @@ TEST_F(MatrixUtilsTest, Circulant) {
   EXPECT_EQ(circulant, expected);
 }
 
+class MatrixPackingTest : public FiniteFieldTest<PackedBabyBear> {};
```

Comment:
I can run this with TypedTest, but I don't think this approach is meaningful. I'm concerned that it will just increase the test running time, and the logic we want to test here doesn't depend on a specific packed prime field implementation. Even though more packed prime fields are introduced, we're unlikely to add them to this test.